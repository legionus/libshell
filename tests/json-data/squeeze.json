{"foo":["bar",47],"asdf":{"qwerty":47,"hooah":47}}
