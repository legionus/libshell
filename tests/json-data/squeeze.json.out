/foo/0 string bar
/foo/1 number 47
/asdf/qwerty number 47
/asdf/hooah number 47
